endpoints.json

```diff
@@ -23,5 +23,77 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/:article_id": {
+    "description": "serves information about a specific article",
+    "queries": ["article_id"],
+    "exampleResponse": {
+      "article_id": 5,
+      "title": "UNCOVERED: catspiracy to bring down democracy",
+      "topic": "cats",
+      "author": "rogersop",
+      "body": "Bastet walks amongst us, and the cats are taking arms!",
+      "created_at": "2020-08-03T13:14:00.000Z",
+      "votes": 0,
+      "comment_count": 2
+    }
+  },
+
+  "PATCH /api/articles/:article_id": {
+    "description": "takes an object for updating information and serves the updated article",
+    "queries": [{ "inc_votes": "numVotes" }],
```

Comment:
Again - the queries appear after a ? in the route e.g. api/articles?sort_by

This wouldnt have any queries